_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,49 @@
+
+
+Even though it’s only June, our HackIllinois Summer team is already well under way in planning the event for next February! We’ve been exploring tons of new possibilities for what the future of our yearly gathering will look like, and fine tuning the things that didn’t go as well last year.
+
+
+Something Old:
+
+If you haven’t already gotten an email, we’d love to hear your feedback from HackIllinois 2018! [link feedback form here] We’re engaging some of our most notable mentors from last year to get their take of the weekend: what they liked and what they would like changed. These mentors are at the heart of our hacker experience, tons of fun to have on campus, and exceptionally talented contributors in their field. If you didn’t get to engage with them at last year’s event, we didn’t do our jobs properly.
+
+2018 also saw some very engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you wouldn’t otherwise have the opportunity to learn. Being able to take a break from your project is something all of our attendees should feel comfortable doing, and we’ll continue to develop these events for the years to come.
+
+
+Something New:
+
+We’re only 8 months, X days, and X hours until check in, but who’s counting? HackIllinois 2019 will be the 6th time Illinois students organize what has become the premier open source event available to undergraduates. We’ve established our mission this year:
+
+To create an environment conducive to education, collaboration, and mentorship within the Open Source community
+
+This year, we’re putting as much emphasis as we can on making this event more than 36 hours of coding. Learning about open source isn’t enough, 2019 will be the immersive experience of teamwork, guidance, and excitement that hackers have come to expect out of our staff. We’ll run interactive workshops, present lively mini-events, and host top industry talent for our weekend in February.
+
+We’ll continue to make more announcements and blog posts too, so stay tuned for updates right here!
+
+
+Something Borrowed:
+
+We’re always open to feedback and new ideas. If you’ve participated in our event or any other hackathon, conference, get-together, Urbana corn festival, ANYTHING where you found a fun or engaging project, activity, or meal, don’t hesitate to suggest it to us. We’re still in the brainstorming phases for the event, and love to hear from you all in general too, so reach out to us! Working together and getting advice from informed individuals is our style. #OpenSourceHack [or whatever our hashtag is]
+
+Let us know on contact@hackillinois.org if you have suggestions, feedback, or just want to share your experiences with us, we’re always happy to hear from you!
+
+
+Something Blue:
+
+Our team is currently hard at work incorporating blue into HackIllinois 2019.
+
```

Comment:
I liked the overall premise of this article, esp. for a first rough draft. Something that caught my attention was the tone difference of this article vs. our other posts (this one is more centered around us/our team than the actual event, at least from what I was able to glean) - thoughts? otherwise, great work Kyle!

Reply:
I agree with you on not talking about the event itself very much here. We're doing a lot of planning right now that may or may not actually happen in February. I figure it would be a little forward to talk about the things we'll have, and instead talk about the things we care about since those will eventually translate into the things we'll have.